### PR TITLE
[UXE-3790] test: add e2mock test for activity history list

### PR DESCRIPTION
**Pull request by aloisio-m-bastian**
### What is the new behavior introduced by this PR?
Add e2mock test to activity history list page
### Does this PR introduce breaking changes?
- [x] No
- [ ] Yes 

### Does it have a link on Figma?

<hr />

#### Make sure your pull request fits the checklist below (when applicable):

- [x] The issue title follows the format: [ISSUE_CODE] TYPE: TITLE
- [x] Commits are tagged with the right word (feat, test, refactor, etc)
- [ ] Application responsiveness was tested to different screen sizes
- [x] Code is formatted and linted
- [x] Tags are added to the PR

#### These changes were tested on the following browsers:
- [x] Chrome
- [ ] Edge
- [ ] Firefox
- [ ] Safari

### CODE REVIEW

File: cypress/e2mock/activity-history.cy.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+describe('Activity History Spec', () => {
+  beforeEach(() => {
+    cy.login()
+  })
+
+  it('should registry when a network list is created', function () {
+
+    //Arrange
+    cy.intercept('POST', '/api/v3/events/graphql', { statusCode: 200, body: fixtures.queryResponse }).as('activityHistoryApi')
+    cy.openProduct('Activity History')
```
Comment (william-tome) [inline cypress/e2mock/activity-history.cy.js:31]:
```suggestion
```

File: cypress/e2mock/activity-history.cy.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+              authorEmail: "teste@gmail.com",
+              accountId: ""
+          },
+      ]
+  }
+  }
+}
+describe('Activity History Spec', () => {
+  beforeEach(() => {
+    cy.login()
```
Comment (william-tome) [inline cypress/e2mock/activity-history.cy.js:24]:
```suggestion
    cy.login()
    cy.openProduct('Activity History')
    
    networkListName = generateUniqueName('NetworkList')
```

File: cypress/e2mock/activity-history.cy.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import generateUniqueName from '../support/utils'
+import selectors from '../support/selectors'
+
+let networkListName = generateUniqueName('NetworkList')
```
Comment (william-tome) [inline cypress/e2mock/activity-history.cy.js:4]:
```suggestion
let networkListName
```

File: cypress/e2mock/activity-history.cy.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+              comment: "-",
+              type: "created",
+              authorName: "teste",
+              authorEmail: "teste@gmail.com",
+              accountId: ""
+          },
+      ]
+  }
+  }
+}
```
Comment (william-tome) [inline cypress/e2mock/activity-history.cy.js:21]: Mover pra dentro da pasta de fixtures

**Review by william-tome — APPROVED** [review]

**Review by william-tome — APPROVED** [review]

**Review by lucasmendes21 — APPROVED** [review]

**Pull request merged**